docs/projects/redemptions.md

```diff
@@ -1,7 +1,27 @@
 ---
 id: redemptions
 sidebar_label: Redemptions
-title: Redemptions
+title: 1Hive Redemptions App
 ---
 
-This needs to be written.
+An Aragon app that enables redeeming organizations tokens for organization assets.
+
+<br>
+
+## What is the Redemptions app?
+
+The redemptions app allows your organization to create a vault that exchanges your native token for an underlying asset. For example, the 1Hive native token is HONEY, but often times organization members will want DAI or ETH. The redemptions vault allows BEEs to exchange the HONEY they earn within the organization for the underlying asset of the vault. It also allows anyone to purchase HONEY by depositing the underlying asset (ETH/DAI) in the vault. 
```

Comment:
```suggestion
The redemptions app allows your organization to create a vault that exchanges your native token for an underlying asset. For example, the 1Hive native token is HONEY, but often times organization members will want DAI or ETH. The redemptions vault allows BEEs to exchange the HONEY they earn within the organization for the underlying asset of the vault. 
```
As discussed in the ideas repo this is going to be the responsability of a new app: `Token Request`

Reply:
Yeah sorry. I got confused on the functionality. Your edit here fixes that though :)